requirements-test.txt

```diff
@@ -1 +0,0 @@
-nose
```

Comment:
why did you delete this? We may have to add more test dependencies in the future, it's always a good idea to have this file.